tests/test_pif.py

```diff
@@ -65,7 +66,17 @@ def test_VASP(self):
             if f != 'OUTCAR':
                 os.unlink(os.path.join('AlNi_static_LDA', f))
 
-        convert([os.path.join('AlNi_static_LDA', 'OUTCAR')])
+        #   Run the conversion, check that it returns some data
+        result = convert([os.path.join('AlNi_static_LDA', 'OUTCAR')])
+
+        found = False
```

Comment:
FYI, you can use this [accessor method](https://github.com/CitrineInformatics/pypif-sdk/blob/develop/pypif_sdk/accessor.py#L3) from pypif-sdk.